code/rdb/rdbstandard.q

```diff
@@ -11,3 +11,8 @@ moveandclear:{[fromNS;toNS;tab]
  if[tab in key fromNS;
   set[` sv (toNS;tab);0#fromNS tab];
   eval(!;enlist fromNS;();0b;enlist enlist tab)]}
+
+\d .
+
+/-set the upd function in the top level namespace
+upd:.rdb.upd
```

Comment:
have you made sure this works? I think `.rdb.upd` is defined in `processes/rdb.q`, which will be loaded _after_ this file, so this may fail. I think `.rdb.upd` will also usually be defined in config somewhere, but the default definition is in `processes/rdb.q` so if a user _doesn't_ have a definition in config, I think this will fail - we should probably error trap this & put a default here to be safe (same as, or maybe instead of, the one in `processes/rdb.q`)